android/Staccato_AN/app/src/main/java/com/woowacourse/staccato/presentation/common/PhotoAttachFragment.kt

```diff
@@ -20,12 +122,45 @@ class PhotoAttachFragment : BottomSheetDialogFragment() {
         return binding.root
     }
 
+    override fun onViewCreated(
+        view: View,
+        savedInstanceState: Bundle?,
+    ) {
+        binding.handler = this
+    }
+
     override fun onDestroyView() {
         super.onDestroyView()
         _binding = null
     }
 
+    override fun onCameraClicked() {
+        // 카메라 열기
+    }
+
+    override fun onGalleryClicked() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            checkPermissionsAndLaunch(READ_MEDIA_IMAGES)
+        } else {
+            checkPermissionsAndLaunch(READ_EXTERNAL_STORAGE)
+        }
+    }
+
+    private fun checkPermissionsAndLaunch(permission: String) {
+        val isPermissionGranted =
+            ContextCompat.checkSelfPermission(
+                requireContext(),
+                permission,
+            ) == PackageManager.PERMISSION_GRANTED
+        if (isPermissionGranted) {
+            launchGallery()
+        } else {
+            requestPermissionLauncher.launch(permission)
+        }
+    }
+
     companion object {
         const val TAG = "PhotoAttachModalBottomSheet"
+        const val SCHEME = "package"
```

Comment:
값이 `"package"`로 고정되어있는 `SCHEME`이니 네이밍을 자세하게 나타내어도 좋을 것 같아요~

```suggestion
        const val PACKAGE_SCHEME = "package"
```